lib/src/layer/polyline_layer.dart

```diff
@@ -112,6 +148,72 @@ class PolylinePainter extends CustomPainter {
     );
   }
 
+  @override
+  bool? hitTest(Offset position) {
+    if (lastHit == null) {
+      return null;
+    }
+
+    final touch = map.pointToLatLng(math.Point(position.dx, position.dy));
+    final origin = map.project(map.center).toOffset() - map.size.toOffset() / 2;
+
+    Polyline? hit;
+
+    outer:
+    for (final p in polylines.reversed) {
+      // TODO: For efficiency we'd ideally filter by bounding box here. However
+      // we'd need to compute an extended bounding box that accounts account for
+      // the stroke width.
+      // if (!p.boundingBox.contains(touch)) {
+      //   continue;
+      // }
```

Comment:
Remove the "TODO" from the comment please. I'd say this as a future performance enhancement and therefore not a "real" todo that would otherwise get highlighted by IDEs.